src/commands.js

```diff
@@ -78,6 +78,60 @@ DeregisterHandler = async (ctx) => {
     await ctx.otherBot.telegram.sendMessage(telegramId, messages.DeregisterSuccess)
 }
 
+ConfirmHandler = async (ctx) => {
+    if (!ctx.isRegistered) {
+        return ctx.reply(messages.NotRegistered)
+    }
+    var person = ctx.person
+    if (person.confirm) {
+        return ctx.reply(messages.AlreadyConfirmed)
+    }
+    await ctx.reply(messages.AskToDoubleConfirm(ctx.isAngel))
+}
+
+DoubleConfirmHandler = async (ctx) => {
+    if (!ctx.isRegistered) {
+        return ctx.reply(messages.NotRegistered)
+    }
+    const model = ctx.model
+    var person = ctx.person
+    if (person.confirm) {
+        return ctx.reply(messages.AlreadyConfirmed)
+    }
+    //TODO: REMOVE THESE TWO LINES
+    person.confirm = true
+    model.saveToStorage()
+
+    const other = ctx.isAngel ? ctx.angel : ctx.mortal
```

Comment:
I think this is already done by middleware? (So it should be available as a field of ctx) If not, we could put this into middleware